### PR TITLE
Delete edit tiles

### PR DESCRIPTION
**Pull request by eloisab17**
Users can delete tiles or edit the text and color of tiles when a tile is held. 

*WARNING* since all users only have the default board at the moment, changes users make to the board persist to the default board, changing for all users.

If you need to reseed the default board after testing, run 

docker-compose exec app php artisan migrate:fresh --seed

Once each user has a copy of the default board instead of the default board itself, the return lines in _onHoldStart and _onHoldEnd can be commented out, and changes will be unique for a user's board.

### CODE REVIEW

File: resources/js/ClickAndHold.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+//code borrowed from: https://www.youtube.com/watch?v=A95mIE2HdcY
```
Comment (peeley) [inline resources/js/ClickAndHold.js:1]: Is this class still needed, if `Board.js` implements the `_onHoldStart` and `onHoldEnd` functions? I can see `app.js` importing this class, but I can't see it being used anywhere

File: resources/js/components/Board.js [hunk 2]
```diff
@@ -101,12 +125,203 @@ export class Board extends React.Component {
...
+  }
+
+  openEditModal = () => {
+    this.setState({
+      editModal: true
+    });
+  }
+
+  closeEditModal = () => {
+    console.log("close edit modal called")
```
Comment (peeley) [inline resources/js/components/Board.js:223]:
```suggestion

```

File: resources/js/components/Board.js [hunk 3]
```diff
@@ -101,12 +125,203 @@ export class Board extends React.Component {
...
+    this.setState({
+      editModal: false,
+    });
+    console.log(this.state.editModal)
```
Comment (peeley) [inline resources/js/components/Board.js:227]:
```suggestion

```

File: resources/js/components/Board.js [hunk 4]
```diff
@@ -101,12 +125,203 @@ export class Board extends React.Component {
...
+          currentBoard: response.data,
+          boardStack: [response.data],
+          loading: false,
+          folderPath: [response.data.name]
+        });
+      });
+  }
+
   render() {
+    console.log("redering board");
```
Comment (peeley) [inline resources/js/components/Board.js:269]:
```suggestion
```

**Comment by peeley** [comment]
also, do you mind fixing the failing tests? looks like they weren't updated to expect the `id`s to be returned for words/folders/boards, you can just add the `id` field to each of the expected value of the asserts

File: resources/js/ClickAndHold.js (hunk 1 above)
Comment (eloisab17) [reply resources/js/ClickAndHold.js:1]: No its no longer needed. I can remove it and add the citation to those specific functions.

File: resources/js/components/Board.js [hunk 5]
```diff
@@ -100,13 +124,201 @@ export class Board extends React.Component {
       <td> { folder} {'>'}</td>
       );
   }
+  //_onHoldStart and _onHoldEnd borrowed from: https://www.youtube.com/watch?v=A95mIE2HdcY
```
Comment (peeley) [inline resources/js/components/Board.js:127]: I feel like we don't really need this citation anymore - the code has changed enough that it's not copied verbatim, and it depends on React functions that the tutorial didn't cover. We can keep it in if you'd like, though

**Pull request merged**